### ISSUE TITLE
[Suggestion] don't update patch version on breaking changes

### ISSUE THREAD

**Issue by ankopainting** [dial-once/node-cache-manager-redis#57]
we did the node thing of setting our version to "^0.2.1" which was fine, until 

```
<    * By default, it caches everything except null and undefined values.

---
>    * By default, it caches everything except undefined values.

```
```
3c3
<   "version": "0.2.4",

---
>   "version": "0.2.5",
39c39
```

This bit us pretty badly (partially due to our deploy process rebuilding for production).  Seems like the rebuild happened just after the change.

Just a friendly FYI so you know that this decision impacted some netizens :)

**Comment by mrister** [comment]
@ankopainting Sorry to hear you had issues. Can you please give us your use case so we can understand a little bit more as we are having some very related conversations on other issues here?

**Comment by jeff-kilbride** [comment]
@ankopainting  Yep, I agree. Sorry for the inconvenience. We didn't have any tests with the `.wrap` method to catch this issue, but I have now added some.

Please see #56 for the current discussion surrounding how to handle this. I would be interested in your input, since you're using `.wrap`.

**Comment by PuKoren** [comment]
Fixed and redeployed in 0.2.6, sorry about that. It was not intended to be a breaking change, we introduced a bug that is now solved.

(As a side note, you should shrinkwrap your project or run tests in your CI system when building with latest packages - shit happens :/)
